backend/src/apiserver/template/v2_template.go

```diff
@@ -77,9 +77,19 @@ func (t *V2Spec) ScheduledWorkflow(modelJob *model.Job) (*scheduledworkflow.Sche
 		}
 	}
 
+	var pipeline_options argocompiler.Options
```

Comment:
Camel casing required to stay consistent with golang variable definition standards - `PipelineOptions`